airtime_mvc/application/Bootstrap.php

```diff
@@ -133,7 +133,11 @@ protected function _initHeadLink()
         $view->headLink()->appendStylesheet($baseUrl . 'css/redmond/jquery-ui-1.8.8.custom.css?' . $CC_CONFIG['airtime_version']);
         $view->headLink()->appendStylesheet($baseUrl . 'css/pro_dropdown_3.css?' . $CC_CONFIG['airtime_version']);
         $view->headLink()->appendStylesheet($baseUrl . 'css/qtip/jquery.qtip.min.css?' . $CC_CONFIG['airtime_version']);
-        $view->headLink()->appendStylesheet($baseUrl . 'css/styles.css?' . $CC_CONFIG['airtime_version']);
+
+        // Don't include style.css on the Radio Page because there are conflicting styles.
+        if ($_SERVER['REQUEST_URI'] != "/") {
+            $view->headLink()->appendStylesheet($baseUrl . 'css/styles.css?' . $CC_CONFIG['airtime_version']);
+        }
```

Comment:
This is a nasty hack. Is there not a better way to remove this explicitly from one page? 

In the long-term, these stylesheets should not even be included in the Bootstrap file. We should have some sort appendDefaultStylesheets() function that's injected in our controllers.